NAMESPACE

```diff
@@ -39,37 +39,33 @@ importFrom(desc,desc)
 importFrom(desc,description)
 importFrom(devtools,build_readme)
 importFrom(devtools,document)
-importFrom(git2r,add)
-importFrom(git2r,ahead_behind)
-importFrom(git2r,branch_create)
-importFrom(git2r,branch_delete)
-importFrom(git2r,branch_get_upstream)
-importFrom(git2r,branch_target)
+importFrom(gert,git_add)
+importFrom(gert,git_ahead_behind)
+importFrom(gert,git_branch)
+importFrom(gert,git_branch_checkout)
+importFrom(gert,git_branch_create)
+importFrom(gert,git_branch_delete)
+importFrom(gert,git_branch_list)
+importFrom(gert,git_commit_id)
+importFrom(gert,git_config)
+importFrom(gert,git_config_set)
+importFrom(gert,git_diff_patch)
+importFrom(gert,git_fetch)
+importFrom(gert,git_info)
+importFrom(gert,git_init)
+importFrom(gert,git_log)
+importFrom(gert,git_ls)
+importFrom(gert,git_push)
+importFrom(gert,git_stat_files)
+importFrom(gert,git_status)
+importFrom(gert,git_tag_create)
+importFrom(gert,git_tag_list)
 importFrom(git2r,branches)
```

Comment:
There are still `git2r` functions in the `NAMESPACE`. We should either fully switch to `gert` or keep using `git2r`.

Reply:
I know, but I haven't found a solution yet for some of the functions.

Reply:
Done in d91a2f5